collectors/googlestackdriver/collector.js

```diff
@@ -16,6 +16,11 @@ const parse = require('@alertlogic/al-collector-js').Parse;
 const AlLogger = require('@alertlogic/al-aws-collector-js').Logger;
 const logging = require('@google-cloud/logging');
 const packageJson = require('./package.json');
+const protoFiles = require('google-proto-files');
+const protoPath = protoFiles.getProtoPath('cloud', 'audit', 'audit_log.proto');
+const root = protoFiles.loadSync(protoPath);
+const AuditLog = root.lookupType('google.cloud.audit.AuditLog');
```

Comment:
this feels like it belongs in a function rather than next to includes

could you please set this up in the `GooglestackdriverCollector.constructor` and make it a property of the collector obj?